mods/tuxemon/l18n/es_ES/LC_MESSAGES/base.po

```diff
@@ -3438,3 +3476,74 @@ msgstr ""
 
 msgid "spyder_flower_sandy_notweesher"
 msgstr "Oh, no importa. Veo que no tienes un Tweesher."
+
+msgid "spyder_route2_roddick1"
+msgstr "Cuando fui emboscado en la duna por SPIGHTER estuve muy asustado."
+
+msgid "spyder_route2_graf2"
+msgstr "Me sirvieron."
+
+msgid "spyder_wayfarer1_bismuth2"
+msgstr "¡Gorrón!"
+
+msgid "spyder_route2_roddick2"
+msgstr "Oye, ¡no parecías arrastrado en absoluto!"
```

Comment:
"creeped" is not "arrastrado" but "asustado". It makes more sense if you look at the previous sentence in the dialog.
```suggestion
msgstr "Oye, ¡no parecías asustado en absoluto!"
```